hedera-node/hedera-app/src/main/java/com/hedera/node/app/bbm/accounts/AccountDumpUtils.java

```diff
@@ -0,0 +1,487 @@
+        try {
+            return fn.apply(a);
+        } catch (final RuntimeException ex) {
+            return missingValue;
+        }
+    }
+
+    /** Given a mapping from field names to both a field extraction function (extract from an account) and a field
+     * formatter (type-specific), produce the formatted form of all the fields given in the mapping.  Can do either of
+     * the `Format`s: CSV or compressed fields.
+     */
+    private static void formatAccountOtherFields(@NonNull final StringBuilder sb, @NonNull HederaAccount a) {
+        final var fieldAccessors = getFieldAccessors(sb, a);
+        for (final var fieldAccessor : fieldAccessors) {
+            final var l = sb.length();
+            formatFieldSep(sb, fieldAccessor.name());
+            if (!fieldAccessor.apply()) {
+                sb.setLength(l);
+                applySwallowingExceptions(fieldAccessor);
+            }
+        }
+    }
+
+    private static <R> boolean applySwallowingExceptions(@NonNull final Field<R> field) {
+        try {
+            return field.apply();
+        } catch (final RuntimeException ex) {
+            return false;
+        }
+    }
+
+    /** A mapping for all account fields that are _not_ of primitive type.  Takes the field name to a `Field`, which
+     * holds the field name, the field extractor ,and the field formatter. And it _is_ a "mapping" even though it isn't
+     * actually a `Map` data structure like the other mappings for primitive typed fields. */
+    @SuppressWarnings({"java:S1452", "java:S2681"})
+    // 1452: generic wildcard types should not be used in return types - yes, but this is a collection of `Field`s
+    // of unrelated types, yet `Object` is not appropriate either
+    // 2681: a complaint about no braces around `then` clause - yep, intentional, and correct
+    // spotless:off
+    private static List<Field<?>> getFieldAccessors(@NonNull final StringBuilder sb, @NonNull final HederaAccount a) {
+        return Stream.of(
+            Field.of("1stContractStorageKey", a::firstContractStorageKey,
+                doWithBuilder(sb, ThingsToStrings::toStringOfByes)),
+            Field.of("accountKey", a::key, doWithBuilder(sb, ThingsToStrings::toStringOfKey)),
+            Field.of("alias", a::alias, doWithBuilder(sb, ThingsToStrings::toStringOfByes)),
+            Field.of("approveForAllNfts", a::approveForAllNftAllowances,
+                doWithBuilder(sb, ThingsToStrings::toStringOfApprovalForAllAllowances)),
+            Field.of("autoRenewAccount", a::autoRenewAccountId,
+                doWithBuilder(sb, ThingsToStrings::toStringOfAccountId)),
+            Field.of("cryptoAllowances", a::cryptoAllowances,
+                doWithBuilder(sb, ThingsToStrings::toStringOfAccountCryptoAllowances)),
+            // Field.of("firstUint256Key", a::getFirstUint256Key, doWithBuilder(sb, ThingsToStrings::toStringOfIntArray)), TODO: how to get?
```

Comment:
Do this via a `java.nio.ByteBuffer` - see `DumpContractStoresSubcommand` L391-392